README.lt.md

```diff
@@ -0,0 +1,28 @@
+[English](README.md) | [日本語](README.ja.md) | [한국어](README.ko.md) | [Bahasa Indonesia](README.in.md) | [Español](README.es.md) | [Deutsch](README.de.md) | [français](README.fr.md) | [العربية](README.ar.md) | [Português Brasileiro](README.pt-br.md) | [Italiano](README.it.md) | [Nederlands](README.nl.md) | [简体中文](README.zh_hans.md) | [繁體中文](README.zh_hant.md) | [Türkçe](README.tr.md) | [বাংলা](README.bn.md)  | [Русский](README.ru.md) | [Polski](README.pl.md) | [Esperanto](README.eo.md) | [Svenska](README.se.md) | [Magyar](README.hu.md) | [Norsk](README.no.md) | [Bahasa Malaysia](README.ms.md) | [ภาษาไทย](README.th.md) | [Tiếng Việt](README.vi.md) | [Česky](README.cz.md) | [ئۇيغۇر](README.ug.md) | [Български](README.bg.md) | [Հայերեն](README.am.md) | [فارسی](README.fa.md) | [Ελληνικά](README.gr.md) | [Lietuviškai](README.lt.md)
+
+# Būkime areštuoti projektas
+
+Japonijoje, moksleivė buvo pagauta policijos dėl to kad ji padėjo nuorodą į vebsaitą su nesibaigiančių "alert" ciklų JavaScripte šitaip:
+
+```js
+while (1) {
+  alert("!");
+}
+```
+
+Susijusios naujienos:
+
+- (Japoniškai) [https://www.sanspo.com/geino/news/20190304/tro19030418410013-n1.html](https://www.sanspo.com/geino/news/20190304/tro19030418410013-n1.html)
+- (Angliškai) [https://www.zdnet.com/article/japanese-police-charge-13-year-old-for-sharing-unclosable-popup-prank-online/](https://www.zdnet.com/article/japanese-police-charge-13-year-old-for-sharing-unclosable-popup-prank-online/)
+
+Šitai matoma kaip "nusikaltimas" Japonijoje. Tapkime kriminalais ir būkime areštuoti!
+
+Mėgaukis savo kriminalinių gyvenimų!
+
+## Kaip būti areštuotu
+
+Tai lengva. Forkink šitą projektą ir brančink tai kaip gh-puslapį. Tai jau padaryta. Būtų veiksmingiau paskleisti URL "https://{tavopaskyra}.github.io/lets-get-arrested" ant socialinio tinklo. Kai paskleisi tai Twitteryje, naudok haštiagą `#letsgetarrested4jscode`.
```

Comment:
"gh-pages" should not be translated to gh-puslapį, GitHub looks for the specific keyword when hosting user sites.

Looks good to me otherwise! :lithuania: